### PR TITLE
create XrInstance

### PR DESCRIPTION
**Pull request by Aki-7**
## Context

Continuation of #3 .

In OpenXR we must create XrInstance first.
https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#instance
> An OpenXR instance is an object that allows an OpenXR application to communicate with an OpenXR runtime. The application accomplishes this communication by calling [xrCreateInstance](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#xrCreateInstance) and receiving a handle to the resulting [XrInstance](https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XrInstance) object.

## Summary

- [x] Define OpenXR context, which will include OpenXR instance, session, etc.
- [x] Create OpenXR instance and store it in the context 

## How to check behavior

1. Build and run with your Oculus Quest.
2. Confirm that no error log happens from "ZEN".
When some error happens, you will see
```log
E/ZEN: <some error log>
```
3. Checkout XrInstance info in the log
```log
I/ZEN: Instance RuntimeName=Oculus RuntimeVersion=43.178.0
```

### CODE REVIEW

File: app/src/main/cpp/openxr-program.cc [hunk 1]
```diff
@@ -1,19 +1,54 @@
...
-  return true;
+  std::ostringstream err;
+  if (XR_FAILED(res)) {
+    err << "XrResult failure [" << to_string(res) << "]" << std::endl;
+    err << "    Origin: " << originator << std::endl;
+    err << std::string() << "    Source: " << source_location;
+  }
+
+  return err.str();
 }
```
Comment (watasuke102) [inline app/src/main/cpp/openxr-program.cc:43]: This function isn't called from another file, so I suggest move this function to the unnamed namespace.

Comment (Aki-7) [reply app/src/main/cpp/openxr-program.cc:43]: I think it's already in an unnamed namespace.

Comment (watasuke102) [reply app/src/main/cpp/openxr-program.cc:43]: Sorry I overlooked

**Review by watasuke102 — APPROVED** [review]

**Comment by Aki-7** [comment]
Thanks for your review!

**Pull request merged**